README.md

```diff
@@ -1,7 +1,8 @@
 # Host Resolver
-A stub DNS resolver that runs on the host machine on Linux, macOS, and Windows. The main goal behind this stub resolver is more robuset handling of domain name resolutions when VPN split tunnle is setup.  
+A stub DNS resolver that runs on the host machine on Linux, macOS, and Windows. The main goal behind this stub resolver is more robuset handling of domain name resolutions when VPN split tunnel is setup.  
+
+![b822615f-af15-4997-981b-53a6f1153d81 sketchpad (1)](https://user-images.githubusercontent.com/10409174/161309152-e048d73f-fbe6-42a2-a409-c29a7de7f03a.jpeg)
```

Comment:
mermaid UI generator online is broken. Didn't want to waste too much time to figure out what is going on.

Reply:
```suggestion
```mermaid
flowchart LR
  subgraph Host
  isp["Public DNS ISP"]
  corp["Corporate VPN"]
  resolver["Host Resolver"]
  isp --- resolver
  corp --- resolver
  end
  subgraph VM
  dns["DNS"]
  container1["container 1"]
  container2["container 2"]
  dns --- container1
  dns --- container2
  end
  resolver --- dns
```
Sorry you'll need to add your own close quote block, GitHub doesn't know how to deal with nested triple-backticks correctly.

Reply:
Thanks Mark!